entries/n/naver.com.json

```
@@ -0,0 +1,22 @@
+{
+  "NAVER": {
+    "domain": "naver.com",
+    "tfa": [
+      "custom-software"
+    ],
+    "custom-software": [
+      "NAVER app"
+    ],
+    "documentation": "https://help.naver.com/service/5640/contents/21127?lang=en",
+    "recovery": "https://help.naver.com/service/5640/contents/21129?lang=en",
+    "categories": [
+      "social",
+      "email",
+      "entertainment",
+      "backup"
```

```suggestion
      "payments"
```

I don't see any backup offering but I do see a payment provider.

---

```suggestion
      "backup",
      "payments"
```
> I don't see any backup offering but I do see a payment provider.

NAVER provides backup service. (mybox.naver.com)

---

Sounds good.